docs/core/event_handler/api_gateway.md

```diff
@@ -853,6 +853,165 @@ You can instruct API Gateway handler to use a custom serializer to best suit you
+to be used with Application Load Balancer.
+
+Below is an example project layout for AWS Lambda Functions using AWS SAM CLI that allows for relative path
+imports (ie: `from .routers import health`).
+
+!!! tip "See in `src/app/main.py`, when including a route we can add a prefix to those routes ie: `prefix="/health"`."
+!!! tip "See in `src/app/routers/health.py`, when adding a child logger we use `Logger(child=True)`."
+
+=== "Project layout"
+
+    ```text
+    .
+    ├── Pipfile
+    ├── Pipfile.lock
+    ├── src
+    │   ├── __init__.py
+    │   ├── requirements.txt      # pipenv lock -r > src/requirements.txt
+    │   └── app
+    │       ├── __init__.py       # this file makes "app" a "Python package"
+    │       ├── main.py           # Main lambda handler (app.py, index.py, handler.py)
+    │       └── routers           # routers module
+    │           ├── __init__.py   # this file makes "routers" a "Python package"
+    │           ├── items.py      # "items" submodule, e.g. from .routers import items
+    │           ├── health.py     # "health" submodule, e.g. from .routers import health
+    │           └── users.py      # "users" submodule, e.g. from .routers import users
+    ├── template.yaml             # SAM template.yml
+    └── tests
+        ├── __init__.py
+        ├── unit
+        │   ├── __init__.py
+        │   └── test_health.py   # unit tests for the health router
+        └── functional
+            ├── __init__.py
+            ├── conftest.py      # pytest fixtures for the functional tests
+            └── test_app_main.py # functional tests for the main lambda handler
+    ```
+
+=== "template.yml"
+
+    ```yaml  hl_lines="22 23"
+    AWSTemplateFormatVersion: '2010-09-09'
+    Transform: AWS::Serverless-2016-10-31
+    Description: Example service with multiple routes
+
+    Globals:
+        Function:
+            Timeout: 20
+            MemorySize: 512
+            Runtime: python3.9
+            Tracing: Active
+            Environment:
+                Variables:
+                    LOG_LEVEL: INFO
+                    POWERTOOLS_LOGGER_LOG_EVENT: true
+                    POWERTOOLS_METRICS_NAMESPACE: MyServerlessApplication
+                    POWERTOOLS_SERVICE_NAME: ServiceName
+
+    Resources:
+        AppFunction:
+            Type: AWS::Serverless::Function
+            Properties:
+                Handler: app.main.lambda_handler
+                CodeUri: src
+                Description: App function description
+                Events:
+                    HealthPath:
+                        Type: Api
+                        Properties:
+                            Path: /health/status
+                            Method: GET
+                    ItemsPath:
+                        Type: Api
+                        Properties:
+                            Path: /items
+                            Method: GET
```

Comment:
Can we keep /users and /health only? 

That would be great to show the feature without distracting them from design questions like: "Shouldn't this be separately, etc. etc."